sematic/plugins/kuberay_wrapper/basic.py

```diff
@@ -0,0 +1,340 @@
+# Standard Library
```

Comment:
Not 100% satisfied with the name "basic." Perhaps "standard" would be better? Suggestions welcome.

Reply:
Standard is better than basic.

Reply:
will rename!